src/main/java/redis/clients/jedis/PipelineBase.java

```diff
@@ -2088,11 +2088,10 @@
   }
   
   @Override
-  public Response<List<StreamPendingEntry>> xpending(byte[] key, byte[] groupname, byte[] start, byte[] end, int count, byte[] consumername){
+  public Response<List<Object>> xpending(byte[] key, byte[] groupname, byte[] start, byte[] end, int count, byte[] consumername){
```

Comment:
@sazzad16 revert?

Reply:
@gkorland Not exactly but nice catch. Keeping the old version while adding a new method.